.github/workflows/precommits.yml

```diff
@@ -0,0 +1,52 @@
+name: Pre-commit checks
+
+on:
+  pull_request:
+    types: [opened, synchronize, reopened]
+    branches: [main, release]
```

Comment:
I know we do it for other repos, but we shouldn't. We actually wanna run precommit checks on any branch. So let's remove this
```suggestion
```

Reply:
Ah I see, thanks for catching this.